src/FX.GrpcModels/Models.fs

```diff
@@ -0,0 +1,27 @@
+﻿namespace GrpcModels
+
+open System
+
+type Message =
```

Comment:
@webwarrior-ws shouldn't this type go away in the commit that removes the hello from the e2e test?

Reply:
There is no such commit.

Reply:
I misread the last commit then; let's remove this from last commit so that e2e test now tests an FX operation (the most basic one) instead of an imaginary hello op

Reply:
Done